app/src/main/python/gogdl/dl/managers/manager.py

```
@@ -0,0 +1,207 @@
+            self.logger.error(f"Game doesn't support content system api, unable to proceed using platform {self.platform}")
+            exit(1)
+
+        # If Linux download ever progresses to this point, then it's time for some good party
+
+        if len(self.builds["items"]) == 0:
+            self.logger.error("No builds found") 
+            exit(1)
+        self.target_build = self.builds["items"][0]
+
+        for build in self.builds["items"]:
+            if build["branch"] == None:
```

Ruff (0.12.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)